config/notifiers.go

```diff
@@ -521,6 +528,49 @@ func (c *SlackConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+// IncidentioConfig configures notifications via incident.io.
+type IncidentioConfig struct {
+	NotifierConfig `yaml:",inline" json:",inline"`
+
+	HTTPConfig *commoncfg.HTTPClientConfig `yaml:"http_config,omitempty" json:"http_config,omitempty"`
+
+	// URL to send POST request to.
+	URL     *SecretURL `yaml:"url" json:"url"`
```

Comment:
Just to confirm, the URL can contain authentication data? In which case, `SecretURL` is correct, otherwise it should be `URL`. I wasn't sure as I also see an `AlertSourceToken`?